src/runners/express.js

```diff
@@ -38,6 +38,44 @@ function createRunner(req, startTime) {
     return expressEvent;
 }
 
+/**
+ * Check if parametered path and absolute path are equal
+ * @param {Array} parameteredPathSplitted parametered splitted path
+ * @param {Array} pathSplitted absolete splitted path
+ * @return {Boolean} true if equal false if not.
+ */
+function checkIfPathsAreEqual(parameteredPathSplitted, pathSplitted) {
+    for (let i = 0; i < pathSplitted.length; i += 1) {
+        if (parameteredPathSplitted[i] !== pathSplitted[i] &&
+            parameteredPathSplitted[i] &&
+            parameteredPathSplitted[i][0] !== ':') {
+            return false;
+        }
+    }
+
+    return true;
+}
+
+/**
+ * Find the parametered path that was called
+ * @param {*} req - express http request object
+ * @return {Object} parametered path if not find
+ * return the last one.
+ */
+function findCalledParameteredPath(req) {
+    let matchedPath = req.route.path[req.route.path.length - 1];
```

Comment:
isn't it better to return null and not set this tag if no path was found?

Reply:
But at Monday case you asked to set the last one if not found